calico-enterprise/operations/ebpf/enabling-ebpf.mdx

```diff
@@ -170,6 +180,14 @@ kubectl patch felixconfiguration default --patch='{"spec": {"bpfKubeProxyIptable
 
 If both `kube-proxy` and `BPFKubeProxyIptablesCleanupEnabled` is enabled then `kube-proxy` will write its iptables rules and Felix will try to clean them up resulting in iptables flapping between the two.
 
+### Disable host conntrack bypass in MKE
+
+If you running MKE, host conntrack should not be bypassed. This can be done with `kubectl` as follows:
```

Comment:
Can we make operator set this flag in MKE mode?  Better to write code than docs!

Reply:
So we're removing everything from this section? ### Diasable host conntrack bypass in MKE

Reply:
@fasaxc Can we remove this step?

Reply:
If we need to retain this, it might function better as a prerequisite.